tidb/watch/disk-usage-balanced.bash

```diff
@@ -0,0 +1,34 @@
+set -euo pipefail
+. "`cd $(dirname ${BASH_SOURCE[0]}) && pwd`/../../helper/helper.bash"
+
+env=`cat "${1}/env"`
+shift
+
+name=`must_env_val "${env}" 'tidb.cluster'`
+shift
+
+pd_leader_id=`must_pd_leader_id "${name}"`
+version=`env_val "${env}" 'tidb.version'`
+if [ -z "${version}" ]; then
+    version=`must_cluster_version "${name}"`
+fi
+
+while [[ true ]]; do
+    region_scores=`tiup ctl:${version} pd -u "${pd_leader_id}" store --jq ".stores[].status.region_score"`
+    region_scores=`echo "${region_scores}" | tr '\n' ','`
+
+    percentage=$(
+python - <<EOF
+region_scores = [${region_scores}]
+m = sum(region_scores) / len(region_scores)
+largest = max(region_scores)
+smallest = min(region_scores)
+print((largest - smallest) / m)
+EOF
+)
+
+    # Can we find a better way?
```

Comment:
We have an algorithm (mean = avg): `(highest - mean) / mean`
But I think the current code could work (because we are detecting "uneven", not measuring "uneven")